packages/react-components/review/stable/react-components.api.md

```diff
@@ -716,10 +723,10 @@ export interface FileDownloadError {
 export type FileDownloadHandler = (userId: string, fileMetadata: FileMetadata) => Promise<URL | FileDownloadError>;
 
 // @beta
-export interface FileMetadata {
-    extension: string;
-    name: string;
-    url: string;
+export type FileMetadata = FileSharingMetadata;
+
+// @beta
+export interface FileSharingMetadata extends BaseFileMetadata {
```

Comment:
Looks like this is a beta type leak? Or please mark these classes as //@public if there are for stable